ansible/configs/ansible-multitier-infra/aapf_lab_vars/aapf_defaults.yml

```diff
@@ -0,0 +1,172 @@
+        from_port: 22
+        to_port: 22
+        protocol: tcp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+
+  - name: HostSG
+    rules:
+      - name: FromHostsgTCP
+        description: "Allow everything from HostSG nodes"
+        from_port: 1
+        to_port: 65535
+        protocol: tcp
+        from_group: HostSG
+        rule_type: Ingress
+
+      - name: FromHostsgUDP
+        description: "Allow everything from HostSG nodes"
+        from_port: 1
+        to_port: 65535
+        protocol: udp
+        from_group: HostSG
+        rule_type: Ingress
+
+  - name: WebserverSG
+    rules:
+      - name: WebSerHTTPPorts
+        description: "HTTP Public"
+        from_port: 80
+        to_port: 80
+        protocol: tcp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+
+      - name: WebSerHTTPsPorts
+        description: "HTTPs Public"
+        from_port: 443
+        to_port: 443
+        protocol: tcp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+
+# Environment Instances
+instances:
+  - name: "control"
+    count: 1
+    unique: true
+    public_dns: true
+    dns_loadbalancer: true
+    floating_ip: true
+    image: "{{ bastion_instance_image | default(image) }}"
+    flavor:
+      ec2: "{{ bastion_instance_type | default(instance_type) }}"
+    tags:
+      - key: "AnsibleGroup"
+        value: "bastions"
+      - key: "ostype"
+        value: "linux"
+      - key: "instance_filter"
+        value: "{{ env_type }}-{{ email }}"
+    rootfs_size: "{{ rootfs_size_bastion | default(50)  }}"
+    security_groups:
+      - BastionSG
+      - TowerSG 
+
+  - name: "tower"
+    count: 1 
+    unqiue: true
+    public_dns: true
+    dns_loadbalancer: true
+    image: "{{ tower_instance_image | default(image) }}"
+    flavor:
+      ec2: "{{ tower_instance_type | default(instance_type) }}"
+    tags:
+      - key: "AnsibleGroup"
+        value: "tower"
+      - key: "ostype"
+        value: "linux"
+      - key: "instance_filter"
+        value: "{{ env_type }}-{{ email }}"
+    security_groups:
+      - TowerSG
+      - DefaultSG
+      - HostSG
+
+  - name: "webserver"
+    count: "{{ node_instance_count | default(2) }}"
+    public_dns: false
+    image: "{{ webserver_instance_image | default(image) }}"
+    flavor:
+      ec2: "{{ webserver_instance_type | default(instance_type) }}"
+    tags:
+      - key: "AnsibleGroup"
+        value: "webservers"
+      - key: "ostype"
+        value: "rhel"
+      - key: "instance_filter"
+        value: "{{ env_type }}-{{ email }}"
+    key_name: "{{key_name}}"
+    security_groups:
+      - HostSG
+      - WebserverSG
+      - DefaultSG
+
+common_packages_el8:
+  - python3
+  - python3-pip
+  - unzip
+  - bash-completion
+  - tmux
+  - bind-utils
+  - wget
+  - git
+  - vim-enhanced
+  - at
+  - gcc
+  - ansible
+
+update_packages: true
+
+rhel_repos:                              # Repositories that will be available in the environment.
```

Comment:
You should only need this if you have `use_content_view: false`. Otherwise, based on the activation key you're using i think you will get these 3 repos anyhow.